README.md

```diff
@@ -148,6 +149,16 @@ Both `BaseCNN` and `BaseLSTM` also have prediction explanation mechanisms that c
 If you have ideas on how to improve the framework to assess text conversation for constructive and inclusive
 communication, we welcome your contributions!
 
+## Running tests
+```python
```

Comment:
Can you add a sentence above to describe what the command does?
simple, "To run all tests" or something similar would be sufficient.